agent/04_agent_prepare_release.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/network.sh
+source $SCRIPTDIR/utils.sh
+source $SCRIPTDIR/validation.sh
+source $SCRIPTDIR/agent/common.sh
+source $SCRIPTDIR/ocp_install_env.sh
```

Comment:
Also need to include here:
source $SCRIPTDIR/oc_mirror.sh

To handle the case where:
export MIRROR_COMMAND=oc-mirror

Otherwise get a failure:
/home/stack/dev-scripts-latest/utils.sh: line 403: setup_oc_mirror: command not found